### PR TITLE
TELECOMMS, YO

### PR DESCRIPTION
**Pull request by FlattestGuitar**
- All headsets have 30 tiles of range
- Telecomms machinery is no longer needed for radios to work
- Static radios now exist. You can't make them for now, but admins can spawn them. 50 tile range.
- Radio Towers also exist. You also can't make these, but you can hook them up to a static radio for a range increase. +50 range for now, we'll see how it works in practice when we have a map.

I seriously need someone to sprite these.

### CODE REVIEW

**Comment by SamHPurp** [comment]
Travis says no. ;)

File: code/game/objects/items/devices/radio/stationary.dm [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	name = "HAM radio"
+	desc = "Of obvious use to colonists, the Hailey-pattern Autonomous Mumblers are widely employed by $CORPORATION NAME. Do not confuse with regular ham radios.\n\
+	It seems you can connect this one to a larger radio tower or a relay.\n\
+	Use <span class='notice'>\":i\"</span> to talk into it."
+	anchored = TRUE
+	w_class = WEIGHT_CLASS_BULKY
+	canhear_range = 7
+	icon_state = "stationary"
+	density = TRUE
+	dog_fashion = null//smh Ian
```
Comment (SamHPurp) [inline code/game/objects/items/devices/radio/stationary.dm]: wat

File: code/game/objects/items/devices/radio/stationary.dm [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	if(istype(I, /obj/item/weapon/wirecutters))
+		if(!open)
+			to_chat(user, "<span class='notice'>You'll have to open it first!</span>")
+			return
+		for(var/obj/item/device/radio/stationary/S in view(1, src))
+			if(!S.connected_relay)
+				S.connected_relay = src
+				S.broadcast_range += amplification
+				connected_radios += S
+				to_chat(user, "<span class='notice'>You connect [S] to [src].")
```
Comment (SamHPurp) [inline code/game/objects/items/devices/radio/stationary.dm]: I've not done a proper review.... but this span needs to close.

**Comment by FlattestGuitar** [comment]
Shitcode; no merge pls

**Pull request closed without merging**